spec/abstract/SchedContext_A.thy

```diff
@@ -366,8 +368,8 @@ definition
   sched_context_bind_tcb :: "obj_ref \<Rightarrow> obj_ref \<Rightarrow> (unit, 'z::state_ext) s_monad"
 where
   "sched_context_bind_tcb sc_ptr tcb_ptr = do
-    set_sc_obj_ref sc_tcb_update sc_ptr (Some tcb_ptr);
     set_tcb_obj_ref tcb_sched_context_update tcb_ptr (Some sc_ptr);
+    set_sc_obj_ref sc_tcb_update sc_ptr (Some tcb_ptr);
```

Comment:
Was there a particular reason this was changed? Just for the refinement proof?

Reply:
Yep, the order is the other way around in Haskell/C